src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -222,6 +223,55 @@ export class TabsTitleControl extends TitleControl {
 		});
 	}
 
+	private updateTabSizing(): void {
+		if (this.tabsContainer) {
```

Comment:
Instead of `if else`, please use `const tabsContainer = assertIsDefined(this.tabsContainer);` as in other places of the file.